### ISSUE TITLE
[typography] Enable font scaling

### ISSUE THREAD

**Issue by akdetrick** [narmi/design_system#1115]
All type should scale via `rem` units. The body font size should be set with `%` or `calc` in a way that allows user-agent settings to set the body font size.

### Development branch:
`feat/fontsize-respects-user-preferences`

### Definition of Done
- [x] Tokens are generated with `rem` units instead of `px`
- [ ] Replace any hard-coded px values with CSS custom properties
- [ ] Beta release of NDS is cut
- [ ] Beta tested in `banking` and appropriate tickets filed to fix any breakage

**Comment by akdetrick** [comment]
released in https://github.com/narmi/design_system/releases/tag/v3.41.0
